src/common/logging.cc

```diff
@@ -90,9 +92,10 @@ void RayLogger_log_event(DBHandle *db,
                          uint8_t *key,
                          int64_t key_length,
                          uint8_t *value,
-                         int64_t value_length) {
-  int status = redisAsyncCommand(db->context, NULL, NULL, "RPUSH %b %b", key,
-                                 key_length, value, value_length);
+                         int64_t value_length,
+                         double time) {
+  int status = redisAsyncCommand(db->context, NULL, NULL, "ZADD %b %s %b", key,
+                                 key_length, std::to_string(time).c_str(), value, value_length);
```

Comment:
Philipp just pointed out that the string we're creating here in the call to `std::to_string(time)` may go out of scope and invalidate the pointer returned by `c_str()` before the call to `redisAsyncCommand`.

We can get around this by creating the string in the line before the call to `redisAsyncCommand`.